mocks/DDrppi_mocks/countpairs_rp_pi_mocks_kernels.c.src

```diff
@@ -1306,28 +1279,19 @@ static inline int countpairs_rp_pi_mocks_fallback_DOUBLE(const int64_t N0, DOUBL
             for(int kbin=nbin-1;kbin>=1;kbin--) {
                 if(sqr_Dperp >= rupp_sqr[kbin-1]) {
                     const int ibin = kbin*(npibin+1) + pibin;
-                    npairs[ibin]++;
-                    if(need_rpavg) {
-                        rpavg[ibin]+=rp;
+                    src_npairs[ibin]++;
```

Comment:
I guess the idea here is that there's no point in making stack buffers, since the passed buffers are already local to the current thread?

Reply:
Yup. I am also considering whether it would be worthwhile to move to a malloc'ed buffer rather than the stack (but there may be side-effects of false sharing under OpenMP with such a malloc'ed ``src_npairs[nthreads][nbins]`` kind of matrix)